script/DeployFactory.s.sol

```diff
@@ -0,0 +1,26 @@
+//SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.18;
+
+import {Script} from 'forge-std/Script.sol';
+import {CatERC20Factory} from '../src/catERC20Factory.sol';
+import {BaseMultiChainDeployer} from '../lib/GeneralisedIncentives/script/BaseMultiChainDeployer.s.sol';
```

Comment:
Please use short notation to import libraries: `GeneralisedIncentives/script/BaseMultiChainDeployer.s.sol` instead of `../lib/GeneralisedIncentives/script/BaseMultiChainDeployer.s.sol`